setup.py

```diff
@@ -317,6 +320,14 @@ def get_long_description():
         packages=find_packages(),
         package_data={
             "": ["images/*", "*.ini"],
+            "envisage.examples": [
+                "demo/*",
+                "demo/*/*",
+                "demo/*/*/*",
+                "demo/*/*/*/*",
+                "demo/*/*/*/*/*",
+                "demo/*/*/*/*/*/*",
```

Comment:
Wait, really? Is there no concise way to do this in a single string?

Reply:
I couldn't find one. If there is a way, I'd love to know!